README.md

```diff
@@ -80,13 +130,43 @@ export const getAnnouncementsByProjectGroup = async value => {
 ```
 This is in addition to the two default "get" functions created. 
 
-## Screenshots
+## Record Transformations
```

Comment:
Nit: do you want to mention the pluralizing of table names here as well?
"Table names are pluralized for the `getAllRecords` and `getRecordsByAttribute` generated functions in `request.js` using a naive check of whether the table name ends in an `s`."